tachyon/zk/transcript/transcript.h

```diff
@@ -12,22 +12,19 @@
 namespace tachyon::zk {
 
 // A 255-bit challenge.
-template <typename AffinePointTy>
+template <typename F>
 class Challenge255 {
  public:
-  using ScalarField = typename AffinePointTy::ScalarField;
-  static_assert(ScalarField::N <= 4);
+  static_assert(F::N <= 4);
```

Comment:
Seeing `F::N` without context might make reader confused.

```suggestion
  static_assert(F::kLimbNums <= 4);
```